include/vsg/utils/ShaderSet.h

```diff
@@ -135,6 +135,18 @@ namespace vsg
     };
     VSG_type_name(vsg::ShaderSet);
 
+    /**
+     * @brief Create a pipeline layout for some sets of a shader set.
+     *
+     * @param shaderSet the shader set
+     * @param defines Set of defines for assembling the bindings in the sets
+     * @param sets Number of sets to use, starting from 0. Defaults to all sets.
+     * @return the vsg::PipelineLayout object.
+     */
+    extern VSG_DECLSPEC ref_ptr<PipelineLayout> makePipelineLayout(ref_ptr<ShaderSet> shaderSet,
```

Comment:
Why call it makePipelineLayout() when similar code use createSomething() and even the doxygen docs says create a pipeline layout?

Reply:
Fair enough. Actually, I wasn't sure whether or not to make it a non-member function, but I prefer that when the function doesn't depend on any private class members.